src/main/resources/templates/vets/vetList.html

```diff
@@ -26,8 +26,7 @@ <h2>Veterinarians</h2>
 
   <table class="table-buttons">
     <tr>
-      <td><a th:href="@{/vets.xml}">View as XML</a></td>
-      <td><a th:href="@{/vets.json}">View as JSON</a></td>
+      <td><a th:href="@{/vets}">View as XML</a></td>
```

Comment:
how about keeping both View as XML and View as JSON by append the format query param in the href.
```
 <td><a th:href="@{/vets?format=xml}">View as XML</a></td>
 <td><a th:href="@{/vets?format=json}">View as JSON</a></td>
```

Reply:
I like the option 4 by @bclozel which we can remain view as json instead of removing it

Reply:
I don't really get the goal behind that feature, besides showing that you can force a content-type without proper content negotiation. That approach is a bit outdated now.

If the goal is to keep that feature like this, the closest thing is indeed to use option 4).

Reply:
I don't think we should keep that feature so I'll remove that link as part of the polish.